Readme.md

```diff
@@ -3,18 +3,30 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1521)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [SimpleData.cs](./CS/SimpleData.cs) (VB: [SimpleData.vb](./VB/SimpleData.vb))
-* [Window1.xaml](./CS/Window1.xaml) (VB: [Window1.xaml](./VB/Window1.xaml))
-* [Window1.xaml.cs](./CS/Window1.xaml.cs) (VB: [Window1.xaml.vb](./VB/Window1.xaml.vb))
-<!-- default file list end -->
-# How to automatically create columns for all fields in a grid's datasource
+# WPF Data Grid - Generate Columns for All Fields in a Data Source
+
+This example uses the [DataControlBase.AutoGenerateColumns](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.DataControlBase.AutoGenerateColumns) property to populate the grid with columns from its data source.
+
+The image below demonstrates the difference between generation modes (**AddNew**, **KeepOld**, and **RemoveOld**). Manually defined columns are highlighted with light yellow, automatically generated — do not have a background.

```

Comment:
`Manually defined columns are highlighted with light yellow, automatically generated — do not have a background.`

What do you mean by "automatically generated"? Can you clarify what is automatically generated?

Reply:
```suggestion
The image below demonstrates the difference between generation modes (**AddNew**, **KeepOld**, and **RemoveOld**). Manually defined columns are highlighted with light yellow, automatically generated columns do not have a background.
```
It's `automatically generated columns`
I wanted to get rid of repetition and make the sentence shorter.

Reply:
```suggestion
The image below demonstrates the difference between generation modes (**AddNew**, **KeepOld**, and **RemoveOld**). Manually defined columns are highlighted in light yellow; automatically generated columns do not have a background color.
```

Reply:
Thanks. I slightly edited your suggestion.